mozdownload/scraper.py

```diff
@@ -816,6 +816,28 @@ def detect_platform(self):
 
         return platform
 
+    def is_build_dir(self, dir):
+        """Return whether or not the given dir contains a build."""
+
+        url = urljoin(self.base_url, self.build_list_regex, dir)
+
+        if self.application in MULTI_LOCALE_APPLICATIONS \
+                and self.locale != 'multi':
+            url = urljoin(url, self.locale)
```

Comment:
@whimboo I will keep it mind, but does this mean I have to change anything in the current PR? I wouldn't...